### DISCUSSION TITLE
e-reader transfert for travel resilience

### DISCUSSION THREAD

**Discussion by Zoyolin** [CatimaLoyalty/Android#926] [Ideas]
Hi, before discovering catima, I had the following need that I think catima could solve without loosing touch with its original goal.
When traveling there are a few barcodes you want to have accessible on you. Generally they'll be stored on your phone. PB: your phone can run out of battery/fail you. A much more simple and resilient device would be the e-reader.
1 use the phone to capture / strip the information from ticket
2 transfert barcode over to the e-reader.
3 delete the file from the ereader.
What do you think?

**Comment by Altonss** [comment]
That could indeed be a cool feature, some kind of "export card to e-reader" functionality?

**Reply by Zoyolin** [reply]
Yes exactly

**Reply by Zoyolin** [reply]
I have a wifi kobo ereader, I suppose there could be a way ..

**Reply by Altonss** [reply]
And how would such a transfer be able to happend? Export an epub file of the selected cards that you store on the smartphond and then can transfer via the way you want (email, usb, app...)?

**Comment by TheLastProject** [comment]
I don't know how your e-reader works but can't you just take a screenshot and share that? It doesn't sound like a feature that would be used so often a more advanced implementation is necessary?
